apps/console/src/features/identity-providers/components/wizards/trusted-token-issuer/trusted-token-issuer-create-wizard.tsx

```diff
@@ -208,17 +207,17 @@ export const TrustedTokenIssuerCreateWizard: FC<TrustedTokenIssuerCreateWizardPr
 
     /**
      * Check whether loop back call is allowed or not.
-     * 
+     *
      * @param value - URL to check.
      */
-    const isLoopBackCall = (value: string): string => 
-        !(URLUtils.isLoopBackCall(value) && commonConfig?.blockLoopBackCalls) 
-            ? undefined 
+    const isLoopBackCall = (value: string): string =>
```

Comment:
Let's do the following improvement here as well if possible.

https://github.com/wso2/identity-apps/pull/4557#discussion_r1389139198

Reply:
https://github.com/wso2/identity-apps/pull/4802/commits/edbbf94f9a9c48a778dee11975927a89e9e3c820